### PR TITLE
fix Parallel dd fails in ppc

### PR DESCRIPTION
**Pull request by PyLearner**
ID: 1341019

### CODE REVIEW

**Comment by PyLearner** [comment]
in ppc, boot up guest with virtio_blk , the disk name in guest is:

for example:
   the system disk will be /dev/vdb
   the disk will be /dev/vda

**Comment by PyLearner** [comment]
run using both virtio_blk and virtio_scsi 
(1/1) smp_16.131072m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.ppc64.io-github-autotest-qemu.autotest.parallel_dd: PASS (792.76 s)

(1/1) smp_16.131072m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.ppc64.io-github-autotest-qemu.autotest.parallel_dd: PASS (723.51 s)

**Comment by qunfang** [comment]
The patch looks good to me, and cases pass for LE guests as well.
PASS 1-smp_16.131072m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.autotest.parallel_dd

PASS 3-smp_16.131072m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.autotest.parallel_dd

Ack'ed by qzhang@redhat.com

File: shared/control/parallel_dd.control [hunk 1]
```diff
@@ -10,12 +10,16 @@ the files system.
...
-if os.path.exists("/dev/sdb"):
-    fs = job.filesystem('/dev/sdb', job.tmpdir)
-elif os.path.exists("/dev/hdb"):
-    fs = job.filesystem("/dev/hdb", job.tmpdir)
-else:
-    fs = job.filesystem("/dev/vdb", job.tmpdir)
+path = glob.glob("/dev/[hsv]d?")
+path.sort()
+fs = path[-1]
+platform = subprocess.check_output("uname -m", shell=True)
```
Comment (xutian) [inline shared/control/parallel_dd.control]: I think better to 
if "ppc" in platform:
      do something
else:
     keep old logical.

File: shared/control/parallel_dd.control [hunk 2]
```diff
@@ -10,12 +10,16 @@ the files system.
...
+if platform not in ("x86_64", "i386"):
+    if not subprocess.check_output("lsscsi"):
```
Comment (xutian) [inline shared/control/parallel_dd.control]: how about virtio_blk ?

File: shared/control/parallel_dd.control (hunk 1 above)
Comment (PyLearner) [inline shared/control/parallel_dd.control]: updated.

File: shared/control/parallel_dd.control (hunk 2 above)
Comment (PyLearner) [inline shared/control/parallel_dd.control]: if use virtio_blk, the system disk will be /dev/vdb ,howerver the disk will be /dev/vda. when use virtio_scsi ,the system_disk will still be /dev/vda.

**Comment by xutian** [comment]
looks ok to me.

**Pull request merged**